<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="heading_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Interpolation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="input_box">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="vector_layer_label">
        <property name="text">
         <string>Input vector layer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsMapLayerComboBox" name="vector_layer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="attribute_label">
        <property name="text">
         <string>Attribute</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsFieldComboBox" name="attribute"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="method_box">
     <property name="title">
      <string>Method settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="interpolation_method_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="interpolation_method_label">
          <property name="text">
           <string>Interpolation method</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="interpolation_method">
          <item>
           <property name="text">
            <string>IDW</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Kriging</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="interpolation_method_pages">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="idw_page">
         <layout class="QFormLayout" name="formLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="power_label">
            <property name="text">
             <string>Power</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsDoubleSpinBox" name="power">
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="kriging_page">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Variogram model</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="variogram_model">
            <item>
             <property name="text">
              <string>Linear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Power</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gaussian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spherical</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Exponential</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Coordinates type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="coordinates_type">
            <item>
             <property name="text">
              <string>Euclidean</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Geographic</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Kriging method</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="kriging_method">
            <item>
             <property name="text">
              <string>Ordinary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Universal</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="output_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="output_raster_settings_layout">
        <item row="1" column="0">
         <widget class="QLabel" name="output_raster_settings_label">
          <property name="text">
           <string>Output raster settings</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="output_raster_settings">
          <item>
           <property name="text">
            <string>From base raster</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Define manually</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QgsFileWidget" name="output_raster_path">
          <property name="storageMode">
           <enum>QgsFileWidget::SaveFile</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="output_raster_path_label">
          <property name="text">
           <string>Output raster path</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="output_raster_settings_pages">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="base_raster_page">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="base_raster_label">
            <property name="text">
             <string>Base raster</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsMapLayerComboBox" name="base_raster"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="define_manually_page">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="define_manually_layout">
            <item row="0" column="0">
             <widget class="QLabel" name="pixel_size_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>62</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Pixel size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QgsDoubleSpinBox" name="pixel_size">
              <property name="maximum">
               <double>10000.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nodata_label">
              <property name="text">
               <string>Nodata</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QgsDoubleSpinBox" name="nodata">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>-999999999.000000000000000</double>
              </property>
              <property name="value">
               <double>-9999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="extent_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>62</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Extent</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QgsExtentGroupBox" name="extent"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btn_layout">
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="run_btn_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="run_btn">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="form_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1000</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
